src/main/kotlin/no/hvl/dat250/app/service/PollService.kt

```diff
@@ -0,0 +1,28 @@
+package no.hvl.dat250.app.service
+
+import no.hvl.dat250.app.exception.PollNotFoundException
+import no.hvl.dat250.app.model.Poll
+
+interface PollService {
+  /**
+   * create a poll
+   */
+  fun createPoll(poll: Poll): Poll
```

Comment:
Send `PollCreateRequest` som første variabel
```suggestion
  fun createPoll(pollRequest: PollCreateRequest): Poll
```